### PR TITLE
regression_metric.QuantileMetric loss bug

### PR DESCRIPTION
**Pull request by amniskin**
The quantile loss function had a bug where instead of returning (1-alpha) * diff it was returning (alpha - 1) * diff. See https://github.com/scikit-learn/scikit-learn/blob/7b136e9/sklearn/ensemble/gradient_boosting.py#L709 for an outside reference.

I also changed the comment on the line above (not really a bug fix, but it scratches my OCD itch)

### CODE REVIEW

**Comment by amniskin** [comment]
That was dumb. My bad. diff is signed.

File: src/metric/regression_metric.hpp [hunk 1]
```diff
@@ -148,7 +148,7 @@ class L2Metric: public RegressionMetric<L2Metric> {
   }
 };
 
-/*! \brief L2 loss for regression task */
+/*! \brief Quantile loss for regression task */
```
Comment (StrikerRUS) [inline src/metric/regression_metric.hpp]: Hi @amniskin !

Can you please create a PR for this typo? Thanks!
